app/models/administrative-unit.js

```diff
@@ -3,8 +3,7 @@ import OrganizationModel from './organization';
 
 export default class AdministrativeUnitModel extends OrganizationModel {
   @belongsTo('administrative-unit-classification-code') classification;
-  @belongsTo('location') municipality;
-  @belongsTo('location') province;
+  @belongsTo('location') scope;
```

Comment:
What's this for?

Reply:
It's the attribute for this 'Werkingsgebied' that is in the detail view.